check-licenses/action.yml

```diff
@@ -1,63 +1,51 @@
-name: "Check licenses"
-description: "Check if package licenses in the current environment are among the accepted ones"
+name: >
+  Check licenses action
+
+description: 
+  Verifies if the licenses of the dependencies installed in the current
+  environment are compliant with PyAnsys guidelines. This action is assumed to
+  be used in its own job step. It clones the project and installs the project with
+  its runtime dependencies.
 
 inputs:
+
+  # Optional inputs
+
   python-version:
-    description: 'Desired Python version.'
+    description: >
+      Python version used for installing and running ``pre-commit``.
     default: '3.10'
     required: false
     type: string
+
   use-python-cache:
-    description: 'Whether to use Python cache or not. Requires a pyproject.toml file.'
+    description: >
+      Whether to use the Python cache for installing previously downloaded
+      libraries. If ``true``, previously downloaded libraries are installed from the
+      Python cache. If ``false``, libraries are downloaded from the PyPI index.
     required: false
     default: true
     type: boolean
-  skip-install:
-    description: 'Skip project installation.'
-    default: false
-    required: false
-    type: boolean
-  target:
-    description: 'Optional target to be passed for the build process.'
-    required: false
-    default: ''
-    type: string
```

Comment:
Same here. Target, though it is not compulsory, it is a good idea to have in case you have additional "versions" of your package to install.

Reply:
The PyAnsys metapackage uses it for example.